data/human/deep missions.txt

```diff
@@ -2752,3 +2752,1850 @@ mission "Stone of our Fathers 5"
+			`	"Now, at base of wall, those pillars, they are our Setstokkr. They are holy relics of our folk and guided us here, to this very wall, long ago. Just as they will guide us once more to our new home. There are eight Setstokkr here, but once there was nine."`
+			`	The Ondurdis turns back to you. "You must find the ninth Setstokkr, the missing pillar. It was dropped in space long ago, and we believe based on where it started it could have only found its way to one of two worlds. It either on Windblain in the Regor system, or Asgard in the Naos system."`
+			`	The Ondurdis offers you a small storage device. "Setstokkr have... unusual properties. This drive contains information that will allow you to track them. Go to these worlds, see if you can find the Setstokkr, and return to us with news. We are eager to know where Skade will send us."`
+			`	The sky glows and the earth rumbles as a shuttle lands not far from the village. "That shuttle will return you to the spaceport and your ship. May Skade guide you." With that The Ondurdis leaves you.`
+			scene "scene/snowvillage"
+			choice
+				`	(Head to the Shuttle)`
+					goto shuttles
+				`	(Take a moment to investigate Goddess-Stone.)`
+					goto dallys
+			label dallys
+			`	You walk toward the park filled with stone cairns, wooden pillars and Goddess-Stone.`
+			`	At this distance you can now see the cairns have name-plates at their bases. Some are so faded and worn you cannot make out the names. Others are easier to read. They appear to be family names. Names like Myrlinga, Baning, and Amalung are the few you can read.`
+			`	Beyond the cairns are the well-carved wooden pillars, or Setstokkr as the Ondurdis called them. They are each over 10 meters tall, and all covered entirely in runic script, images, and patterns. A few have unshorn branches that include their own carvings as well, while most are limbless. You walk up to one and notice a frequent carving within its entwined animals and shapes is similar to burn on your arm.`
+			`	Finally, you reach the massive Goddess-Stone itself. You see now almost no part of the wide surface is bare. There are images within images, letters within letters. These exotic letters bear no relationship to the runes found on the Setstokkr. These are truly alien. `
+			choice
+				`	(You've seen enough, and there is much that needs to be done. You head to the shuttle.)` 
+					goto shuttles
+			label shuttles			
+			`	A lone figure stands next to the idling shuttle. As you draw closer you recognize Hjlod.`
+			`	"You see!" Hjlod says, "Stick with Hjlod, and you live, probably." She holds up a bandaged three fingered hand and shrugs. "Mostly."`
+			`	Hjlod offers one of her lopsided grins and then leaves you alone with the shuttle. It is time to go.`
+				accept
+	on stopover
+		dialog
+			`	You land on the final world you were asked to visit. Once you're done here you should return to Nifel.`
+	on complete
+		event "Nifelwaiting" 15
```

Comment:
```suggestion
		event "nifel waiting" 15
```
Events can have spaces in them, but should be lowercase by default (which most of yours are, just for this one in particular). The spaces will help make them a bit more readable in the files.